stripe/src/test/java/com/stripe/android/StripeTest.java

```diff
@@ -248,6 +253,204 @@ public void createBankAccountTokenSynchronous_withValidBankAccount_returnsToken(
     }
 
     @Test
+    public void createSourceSynchronous_withBitcoinParams_passesIntegrationTest() {
+        Stripe stripe = new Stripe(mContext);
+        SourceParams bitcoinParams = SourceParams.createBitcoinParams(1000L, "usd", "abc@def.com");
```

Comment:
not sure if you support this right now, but want to note that you should also be able to attach metadata when creating a source [0] – this is important since a user may want to tag the source with order information.

[0] https://github.com/stripe/stripe-ios/blob/master/Tests/Tests/STPSourceFunctionalTest.m#L26